internal/ingress/controller/controller.go

```diff
@@ -957,12 +964,14 @@ func (n *NGINXController) createServers(data []*ingress.Ingress,
 	}
 
 	// initialize default server and root location
+	ptf := networking.PathTypePrefix
```

Comment:
Why `ptf`but not `ptp`? But generally I'd rather not shorten variable names like this at all and would go with `pathTypePrefix` instead.

Reply:
done